data/json/effects.json

```diff
@@ -1456,5 +1456,33 @@
         "scaling_mods": {
             "hurt_chance": [1]
         }
+    },
+    {
+        "type": "effect_type",
+        "id": "skin_rot",
+		"duration_decay": 1,
+		"max_duration": 57599,
+		"max_intensity": 4, 
+		"int_dur_factor": 14400,
+        "name": ["Itchy skin", "Thinning skin", "Dermal degeneration", "Dermal degeneration"],
+        "desc": ["Your skin feels itchy.", "Layer of your skin peel away.", "Your thin skin is cracked.", "Your thin skin is marred by deep cracks."],
+        "rating": "bad",
+		"scaling_mods": {
+            "armor_cut_mod": [-1]
```

Comment:
Cutting armor isn't really a good thing to mod right now.
Pretty much the only two sources that do decent amount of cutting damage are firearms and armed NPCs.